### ISSUE TITLE
Android background service

### ISSUE THREAD

**Issue by vojtechkral** [mapsme/omim#455]
I noticed there's a Maps.me background service that starts on boot.
What does this service do? Could it be made configurable?

**Comment by deathbaba** [comment]
Now it is not yet configurable. It checks if maps are downloaded for region where you are now, and recommends to download them. Also it checks if map updates are available.

**Comment by vojtechkral** [comment]
Okay, thanks for info. Would you be open to the idea of making this configurable?

**Comment by deathbaba** [comment]
Yes, it is a good idea and needs to be discussed. In general, we try to avoid many different settings and satisfy 99% of users automatically. OSMand is an opposite example which we don’t want to follow to stay simple.

May be it can be easier to incorporate it in some “hidden” advanced settings.

**Comment by RicoElectrico** [comment]
Yeah, but consider that many view unsolicited notifications as a nuisance. Do I have to mention that fail when you pushed "car free day" notifications in every single language?
![screenshot_2015-09-21-16-45-32](https://cloud.githubusercontent.com/assets/1409116/11031595/f52a9322-86d5-11e5-9656-b9c117feb18e.png)


As per OsmAnd, it's not even about its functionality, but about its clunky interface. If you want to distance yourself from it, at least know why. Google/Apple Maps have very sophisticated features, but they're behind a simple and easy-to-use interface.

**Comment by vojtechkral** [comment]
Well, the setting could probably be described in simple terms. Such as, for example, _"Check for map updates automatically"_ or something along those lines...

**Comment by deathbaba** [comment]
Push notifications bug is not related at all to discussed settings (and discussed setting would not stop it even if it existed).

**Comment by deathbaba** [comment]
We'll discuss it with our team, but it's definitely not a top priority for us right now. You can, however, add it in your own Pull Request :)

**Comment by vojtechkral** [comment]
Okay :)
